src/sentry/api/bases/organization_events.py

```diff
@@ -293,7 +269,7 @@ def handle_results_with_meta(
                     "datasetReason": meta.get("datasetReason", discover.DEFAULT_DATASET_REASON),
                 }
                 if dataset is not None:
-                    meta["dataset"] = DATASET_LABELS.get(dataset, "unknown")
+                    meta["dataset"] = snuba.DATASET_LABELS.get(dataset, "unknown")
```

Comment:
I think you may have forgotten to port over the `DATASET_LABELS` constant? I can't see it in this PR and it doesn't seem to already be in the `snuba` import

Reply:
:+1: Yup fixed in e831801